std_msgs/Header header
float64 depth_rel_reader             # depth_rel_reader reading
float64 depth_rel_reader_variance    # depth_rel_reader_variance reading
